bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Region.java

```diff
@@ -107,13 +104,8 @@ public Region (Device device) {
 public void add (int[] pointArray) {
 	if (isDisposed()) SWT.error(SWT.ERROR_GRAPHIC_DISPOSED);
 	if (pointArray == null) SWT.error(SWT.ERROR_NULL_ARGUMENT);
-	addInPixels(DPIUtil.autoScaleUp(pointArray));
-}
-
-void addInPixels (int[] pointArray) {
-	long polyRgn = OS.CreatePolygonRgn(pointArray, pointArray.length / 2, OS.ALTERNATE);
-	OS.CombineRgn (handle, handle, polyRgn, OS.RGN_OR);
-	OS.DeleteObject (polyRgn);
+	final Operation operation = new OperationWithArray(Operation::add, Arrays.copyOf(pointArray, pointArray.length));
```

Comment:
That's nice 👍  I was not aware that it is possible to pass a method reference to an abstract method that is properly the bound in the context of the instantiated subclass.